lib/overcommit/git_repo.rb

```diff
@@ -272,5 +272,23 @@ def branches_containing_commit(commit_ref)
     def current_branch
       `git symbolic-ref --short -q HEAD`.chomp
     end
+
+    # Returns the names of the files that have been modified comparing the
+    # current local branch with the corresponding remote one if it does exist or
+    # the current local branch with the parent one if not.
+    # @param remote [String] name of the remote
+    #
+    # @return [Array<String>] list of relative file paths
+    def modified_file_in_branch(remote)
+      remote_url = `git remote get-url #{remote}`.split("\n").first
```

Comment:
There's a danger here with using backticks as any failures are silently ignored (potentially returning an empty string). 

This isn't a big deal but if you have time it might be worth using the `execute` helper so you can actually inspect the exit status of the process. Not a blocker.